src/components/BuySell/BuySellWidget.tsx

```diff
@@ -1,26 +1,38 @@
-import React from 'react'
+import React, { ComponentProps } from 'react'
 import styled from 'styled-components'
 
 import BuySellSelector from './components/BuySellSelector'
 import TokenInputs from './components/TokenInputs'
 import OrderSummary from './components/OrderSummary'
 import BuySellButton from './components/BuySellButton'
 import HowToBuyLink from './components/HowToBuyLink'
+import ExchangeSelector from './components/ExchangeSelector'
 
-const BuySellWidget: React.FC = () => {
+interface Props {
+  setIsIssuance: any
+}
+
+const BuySellWidget: React.FC<Props> = ({ setIsIssuance }) => {
   return (
-    <StyledBuySellCard>
-      <StyledBuySellCardContent>
-        <BuySellSelector />
-        <TokenInputs />
-        <OrderSummary />
-        <BuySellButton />
-        <HowToBuyLink />
-      </StyledBuySellCardContent>
-    </StyledBuySellCard>
+    <StyledContainer>
+      <StyledBuySellCard>
+        <StyledBuySellCardContent>
+          <BuySellSelector />
+          <TokenInputs />
+          <OrderSummary />
+          <BuySellButton />
+          <HowToBuyLink />
+        </StyledBuySellCardContent>
+      </StyledBuySellCard>
+      <ExchangeSelector setIsIssuance={setIsIssuance} />
```

Comment:
ExchangeSelector shouldn't be available on all buy/sell widgets. Only on CGI + DPI. INDEX buy sell widget should not allow exchange issuance

Reply:
fixed by wrapping exchange Selector outside buysell widget